index.js

```diff
@@ -11,16 +11,47 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+const maxPage = 42;
+let page = 1;
 const searchQuery = "";
 
 // Functions
 
 async function fetchCharacters() {
-  const response = await fetch("https://rickandmortyapi.com/api/character");
+  const response = await fetch(
+    `https://rickandmortyapi.com/api/character?page=${page}`
+  );
   const json = await response.json();
   return json.results;
 }
+
+function pageChange() {
+  const pageChange = `<span class="navigation__pagination" data-js="pagination">${page} / ${maxPage}</span>`;
+  pagination.innerHTML = pageChange;
+}
+
+pageChange();
 const characters = await fetchCharacters();
 createCharacterCard(characters);
+
+prevButton.addEventListener("click", async (event) => {
+  if (page === 1) {
+    event.target.disabled = true;
+  } else {
+    --page;
+    const characters = await fetchCharacters();
+    createCharacterCard(characters);
+    pageChange();
+  }
+});
```

Comment:
If you follow my advise above, this function does not need to be async.
Also you would not need to call `pageChange()` here. You can simply to it inside the fetch function, so you do not need to do it twice (for each button).

You can disable the button if you like. But it is not really necessary here. If you only subtract 1 when `page > 1`, the page won't be less than that and further button clicks will not have any effect.
```suggestion
prevButton.addEventListener("click", () => {
  if (page > 1) {
    --page;
    fetchCharacters();
  }
});
```

Reply:
This is not updated yet 🤔